src/index.html

```diff
@@ -5,13 +5,164 @@
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    ><meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="personal-Information">
+        <legend>Personal information</legend>
+        <label class="formField">
+          Surname:
+          <input
+            class="input"
+            type="text"
+            name="userSurename"
```

Comment:
kebab-case is required for attribute names, not camelCase. `here-is-the-example-of-text-written-with-kebab-case`

Reply:
disregard this one, I was not right